README.md

```diff
@@ -22,6 +22,86 @@ Add the following code to the `pom.xml` file of the project where you want to ex
 		<version>1.0-SNAPSHOT</version>
 		<configuration>
 			<agentPath>{path_to_opentelemetry-javaagent-all.jar}</agentPath>
+
+			<exporter>
+				<otlpExporter>
+					<tracesExporter>otlp</tracesExporter>
+					<metricExporter>otlp</metricExporter>
+
+					<exporterOtlpEndpoint></exporterOtlpEndpoint>
+					<exporterOtlpTracesEndpoint></exporterOtlpTracesEndpoint>
+					<exporterOtlpMetricsEndpoint></exporterOtlpMetricsEndpoint>
+
+					<exporterOtlpHeaders></exporterOtlpHeaders>
+					<exporterOtlpTracesHeaders></exporterOtlpTracesHeaders>
+					<exporterOtlpMetricsHeaders></exporterOtlpMetricsHeaders>
+
+					<exporterOtlpTimeout></exporterOtlpTimeout>
+					<exporterOtlpTracesTimeout></exporterOtlpTracesTimeout>
+					<exporterOtlpMetricsTimeout></exporterOtlpMetricsTimeout>
+				</otlpExporter>
+				<jaegerExporter>
+					<tracesExporter>jaeger</tracesExporter>
+
+					<exporterJaegerEndpoint></exporterJaegerEndpoint>
+					<exporterJaegerTimeout></exporterJaegerTimeout>
+				</jaegerExporter>
+				<zipkinExporter>
+					<tracesExporter>zipkin</tracesExporter>
+
+					<exporterZipkinEndpoint></exporterZipkinEndpoint>
+				</zipkinExporter>
+				<prometheusExporter>
+					<metricExporter>prometheus</metricExporter>
+
+					<prometheusPort></prometheusPort>
+					<prometheusHost></prometheusHost>
+				</prometheusExporter>
+				<loggingExporter>
+					<tracesExporter>logging</tracesExporter>
+					<metricExporter>logging</metricExporter>
+
+					<exporterLoggingPrefix></exporterLoggingPrefix>
+				</loggingExporter>
+			</exporter>
+
+			<propagator>
+				<propagator>tracecontext</propagator>
+				<propagator>baggage</propagator>
+			</propagator>
+
+			<openTelemetryResource>
+					<attributes>
+						<key1>value1</key1>
+						<key2>value2</key2>
+					</attributes>
+					<serviceName></serviceName>
+			</openTelemetryResource>
+
```

Comment:
In a recent commit, they added something called 'Disabling Automatic ResourceProviders'. We can add this too.